scripts/serial_link.py

```diff
@@ -382,6 +382,30 @@ def format_log_json_entry(t0, item):
   data = item.to_json_dict()
   return {'delta': delta, 'timestamp': timestamp, 'data': data}
 
+class Watchdog:
+  def __init__(self, timeout, alarm):
```

Comment:
nit: a brief description, possibly saying that it calls an alarm raising a `KeyboardInterruptException` (?) or something after timeout.